packages/clay-navigation/src/Vertical.tsx

```diff
@@ -21,6 +21,11 @@ interface IItem {
 	 */
 	onClick?: () => void;
 
+	/**
+	 * Object of props for an item.
+	 */
+	itemProps?: Object;
```

Comment:
Why an exclusive prop? where the element is the parent of the component.

Reply:
Solved after some internal conversation!